Frameworks/CoreMotion/CMMotionManager.mm

```diff
@@ -15,35 +16,149 @@
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 - (void)startAccelerometerUpdatesToQueue:(NSOperationQueue*)queue withHandler:(CMAccelerometerHandler)handler {
-    UNIMPLEMENTED();
+    
+    if (self.accelerometerAvailable) {
+        [self startAccelerometerUpdates];
+        self.accelerometerQueue = queue;
+        [self setAccelerometerReportInterval];
+
+        self.accelerometerToken = [self.accelerometer addReadingChangedEvent:
+            ^void(WDSAccelerometer* sender, WDSAccelerometerReadingChangedEventArgs* e) {
+
+                CMAccelerometerData* data = [[CMAccelerometerData alloc] initWithValues:e.reading.accelerationX 
+                                                                                      y:e.reading.accelerationY
+                                                                                      z:e.reading.accelerationZ];
+                [queue addOperationWithBlock:^{ handler(data, nil); }];                                   
+            }];          
+    }    
 }
 
+
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 - (void)startAccelerometerUpdates {
-    UNIMPLEMENTED();
+    
+    if (self.accelerometerAvailable) {
+        
+        if (self.accelerometerActive) {
+            [self stopAccelerometerUpdates];
+        }
+        
+        self.accelerometerActive = true;
+    }        
 }
 
+
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
-- (void)stopAccelerometerUpdates {
-    UNIMPLEMENTED();
+- (void) stopAccelerometerUpdates {
+
+    if (self.accelerometerActive) {
+        self.accelerometerActive = false;
+        
+        // the reportInterval is set to 0, the default value.
+        self.accelerometer.reportInterval = 0;
+        
+        if (self.accelerometerToken.value != 0) {
+            [self.accelerometer removeReadingChangedEvent:self.accelerometerToken];
+            self.accelerometerToken = {0};
+        }
+        
+        if (self.accelerometerQueue) {
+            [self.accelerometerQueue cancelAllOperations];
+            self.accelerometerQueue = nil;
+        }
+    }                         
+ }
+
+
+// getter function for on-demand reading from sensor
+ -(CMAccelerometerData*)accelerometerData {
+    
+    WDSAccelerometerReading* reading = [self.accelerometer getCurrentReading];
+
+    _accelerometerData = [[CMAccelerometerData alloc] initWithValues:reading.accelerationX 
+                                                                   y:reading.accelerationY
+                                                                   z:reading.accelerationZ];
+    return _accelerometerData;
+ }
+
+
+-(void)setAccelerometerReportInterval {
+    self.accelerometer.reportInterval = static_cast<unsigned int>(self.accelerometerUpdateInterval * 1000);
 }
 
+
+
+-(void)setAccelerometerUpdateInterval:(NSTimeInterval)updateInterval {
+    @synchronized(self) {
+        
+        if (updateInterval * 1000 < self.accelerometer.minimumReportInterval) {
```

Comment:
Can you add comments for the conversion here? Something simple like 'IOS uses seconds whereas WinRT uses milliseconds' would be helpful.  Bonus points for pulling 1000 out as a constant that is descriptively named.